gossip/hooks.py

```diff
@@ -123,6 +123,7 @@ def trigger(self, kwargs, tags=None):
         if self._unmet_deps:
             raise CannotResolveDependencies('Hook {0!r} has unmet dependencies: {1}'.format(self, ', '.join(map(str, self._unmet_deps))))
         if self.full_name in _muted_stack[-1]:
+            _logger.debug("Hook {!r} muted, skipping trigger", self)
```

Comment:
Not sure how this passed the tests, but both Travis and tox also run it through 2.6...